rastervision_core/rastervision/core/data/raster_source/multi_raster_source.py

```diff
@@ -148,11 +167,26 @@ def get_chip(self, window: Box) -> np.ndarray:
         Returns:
             np.ndarray with shape [height, width, channels]
         """
-        chip_slices = [rs.get_chip(window) for rs in self.raster_sources]
+        if self.all_extents_equal:
```

Comment:
The code in `get_chip` and `_get_chip` are almost identical. Consider extracting the common code to a function.

Reply:
Good point. Fixed.